requirements.txt

```diff
@@ -5,4 +5,5 @@ pure-python-adb==0.3.0.dev0
 Flask-SocketIO==5.3.6
 protobuf>=4.24.2
 pyaxmlparser==0.3.30
-up-python==0.1.2.dev0
\ No newline at end of file
+up-python==0.1.2.dev0
```

Comment:
the semver standard we use would be more 0.1.2-dev, is the "." and the "0" required by pypi?

Reply:
Oh, I wasn't aware of this standard, the '0' isn't necessary, as it works with the 0.1.2-dev version too. I'll fix it